packages/angular-sample/package.json

```diff
@@ -21,7 +21,7 @@
     "@angular/platform-server": "^19.0.0",
     "@angular/router": "^19.0.0",
     "@angular/ssr": "^19.0.0",
-    "@logto/js": "link:../js",
+    "@logto/angular": "link:../angular",
```

Comment:
use `workspace:^`?

Reply:
Oh, the sample project is not a pnpm package, isn't it. If it's not, then OK to use `link`

Reply:
Yes, this package is special